### PR TITLE
[SDPA-4462] update ripple to display multiple contacts in sidebar

### PR DESCRIPTION
**Pull request by rubyllamadora**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4462

### Changed

1.  Updated sidebar to accept multiple contact data

### Related PR
https://github.com/dpc-sdp/tide_core/pull/165 - backend PR by Vincent Gao

### Screenshots

Screen Shot 2020-08-24 at 9 55 26 am

### CODE REVIEW

**Review by tim-yao — APPROVED** [review]

**Comment by tim-yao** [comment]
Hey @rubyllamadora Looks like this BE/FE change, caused the e2e test failed.
```
1) Demo Page Demo landing page loads correctly:
   CypressError: Timed out retrying: Expected to find element: '.rpl-contact', but never found it.
```
Can you please have a look at it? Thanks!

**Comment by rubyllamadora** [comment]
> Hey @rubyllamadora Looks like this BE/FE change, caused the e2e test failed.
> 
> ```
> 1) Demo Page Demo landing page loads correctly:
>    CypressError: Timed out retrying: Expected to find element: '.rpl-contact', but never found it.
> ```
> 
> Can you please have a look at it? Thanks!

Thanks for review Tim. I ran smoke test on my local and it passed. It's the BE changes that is not yet in develop which does not support multiple contacts yet. I'll wait for @vincent-gao to merge his changes to develop and will test this again.

**Comment by vincent-gao** [comment]
> > Hey @rubyllamadora Looks like this BE/FE change, caused the e2e test failed.
> > ```
> > 1) Demo Page Demo landing page loads correctly:
> >    CypressError: Timed out retrying: Expected to find element: '.rpl-contact', but never found it.
> > ```
> > 
> > 
> > Can you please have a look at it? Thanks!
> 
> Thanks for review Tim. I ran smoke test on my local and it passed. It's the BE changes that is not yet in develop which does not support multiple contacts yet. I'll wait for @vincent-gao to merge his changes to develop and will test this again.

Hi @rubyllamadora , could you please point the FE to https://nginx-php-content-vic-pr-933.lagoon.vicsdp.amazee.io/

**Comment by rubyllamadora** [comment]
> > > Hey @rubyllamadora Looks like this BE/FE change, caused the e2e test failed.
> > > ```
> > > 1) Demo Page Demo landing page loads correctly:
> > >    CypressError: Timed out retrying: Expected to find element: '.rpl-contact', but never found it.
> > > ```
> > > 
> > > 
> > > Can you please have a look at it? Thanks!
> > 
> > 
> > Thanks for review Tim. I ran smoke test on my local and it passed. It's the BE changes that is not yet in develop which does not support multiple contacts yet. I'll wait for @vincent-gao to merge his changes to develop and will test this again.
> 
> Hi @rubyllamadora , could you please point the FE to https://nginx-php-content-vic-pr-933.lagoon.vicsdp.amazee.io/

Hi @vincent-gao, doing it now I just figured out :D It has pass demo landing page and it returns another error. Looking into this now. Please ignore message above.

File: examples/vic-gov-au/nuxt.config.js
Comment (rubyllamadora) [inline examples/vic-gov-au/nuxt.config.js:121]: Note to self: Revert this before merge.

File: packages/ripple-nuxt-tide/lib/middleware/sidebar.js
Comment (rubyllamadora) [inline packages/ripple-nuxt-tide/lib/middleware/sidebar.js:121]: Hi @tim-yao please review updates on validation (:

**Review by tim-yao — APPROVED** [review]

**Pull request merged**
